CASMTRIAGE-7189 Update IUF.md (#5281)
Cray-HPE/docs-csm: operations/iuf/IUF.md

Argo workflows based on the subcommand specified. The Argo workflows are not controlled by `iuf` once they have been created, but
`iuf` does display status to the administrator as the Argo workflows execute.

**`NOTE`** Before starting, [Validate CSM Health](../validate_csm_health.md). Ensure sufficient memory, CPU and disk usage

> is available (example commands given below as reference only, do change accordingly based on number of products used by IUF).

Reviewer: ```suggestion
is available (the following example commands are for reference only; change them accordingly based on number of products used by IUF).
```

(`ncn-m001#`) List processors and disk usage using built-in commands `nproc` and `df`.

```bash
nproc --all
df -h /
```

Example output:

```text

> nproc --all
> 40
> df -h /
> Filesystem      Size    Used     Avail   Use%  Mounted on
> LiveOS_rootfs   <N>G    <M>G     <N-M>G  45%   /

Reviewer 2: ```suggestion
# nproc --all
40
# df -h /
Filesystem      Size    Used     Avail   Use%  Mounted on
LiveOS_rootfs   <N>G    <M>G     <N-M>G  45%   /
```
